gtk/src/Makefile.am

```diff
@@ -179,7 +179,7 @@ widget.deps: makedeps.py
 	$(HB_PYTHON) $(srcdir)/makedeps.py
 
 fr.handbrake.ghb.metainfo.xml: fr.handbrake.ghb.metainfo.xml.in
-	$(AM_V_GEN) $(MSGFMT) --xml --template $< -d $(top_srcdir)/po -o $@
+	$(AM_V_GEN) $(MSGFMT) --xml --template $< -d $(top_srcdir)/po -o $@ || cp $< $@
```

Comment:
This looks odd.

Reply:
This is a workaround for NetBSD, which has an old version of gettext which doesn't support xml. If `msgfmt` fails, it just copies the file without adding the localized strings. It was the best solution I could come up with.

Reply:
Please add an inline code comment explaining why `cp` is used here.

Reply:
Done.